global-elements/heroes.md

```diff
@@ -33,75 +32,183 @@ Heroes function as a primary focal point on a page, often used to introduce a co
 </div> 
 
 
-<h2 id="use">Use<span class="cf-code-link"><a href="https://github.com/cfpb/capital-framework/blob/master/src/cf-layout/src/cf-layout.less#L618-L620">View code <span class="cf-icon cf-icon-external-link"></span></a></span></h2>
-
+<h2 id="use-case">Use case<span class="cf-code-link"><a href="https://github.com/cfpb/capital-framework/blob/master/src/cf-layout/src/cf-layout.less#L618-L620">View code <span class="cf-icon cf-icon-external-link"></span></a></span></h2>
 
 <div class="content-67 content-first">
-Content should be limited to one or two brief sentences to help the user orient themselves and ensure this page will satisfy their desired goal. Copy should be punchy and grab the user’s attention while reinforcing the brand voice. 
+Heroes are the most prominent component on a page, in terms of their position and visual footprint. Heroes should only be used on top level pages, including landing and sublanding pages.
+</div>
 
-Heroes are integral to the page, and cannot be removed. 
+<div class="content-33 content-last">
+##### Where to use
+[Landing page]({{site.baseurl}}/global-elements/landing-page.html)
 
-Heroes should be limited to first- (i.e., Consumer Help) or second-level (i.e., Credit & Credit Cards) landing pages. The fewer instances of heroes there are, the more impact they will convey. 
+[Sublanding page]({{site.baseurl}}/global-elements/sublanding-page.html)
+</div>
 
-Heroes should be the most prominent thing on the page, and help to establish visual hierarchy of where the page lives within the information architecture.
+<img alt="Image of hero graphic" src="../static/img/hero/hero_use_case.png"/>
```

Comment:
We've preferred using absolute urls, which would update the source here to `{{site.github.urll}}/static/img/hero/hero_use_case.png` (We should update all the existing `{{site.baseurl}}` uses but that's outside the scope of this PR)